### PR TITLE
turn member list into a table with a GitHub column and a Twitter column

### PR DESCRIPTION
**Pull request by amor520**
turn member list into a table with a GitHub column and a Twitter column

image

### CODE REVIEW

**Comment by amor520** [comment]
#39

**Comment by amor520** [comment]
image

File: README.md [hunk 1]
```diff
@@ -45,24 +45,27 @@ In order to ensure best practices, this working group may work directly with rep
...
-  + [Steven R. Loomis](https://twitter.com/srl295)
-  + [Tiago Danin](https://twitter.com/_TiagoEDGE)
-  + [Vanessa Yuen](https://twitter.com/vanessayuenn)
-  + [Volkan Nazmi Metin](https://twitter.com/volemnic)
-  + [Zeke Sikelianos](https://twitter.com/zeke)
+| Name                  | GitHub                        | Twitter                             |
+|:----------------------|:------------------------------|:------------------------------------|
+| Adam Miller           |                               | https://twitter.com/millea9         |
+| Antoine Olivier       |                               | https://twitter.com/Toinane         |
+| Amor                  | https://github.com/amor520    | https://twitter.com/amor90s         |
```
Comment (RichardLitt) [inline README.md]: I think this would work the same as a link on GitHub rendered markdown if you just use @amor520 instead of putting in the whole link.

**Comment by RichardLitt** [comment]
Looks pretty good. Why not use `[@richlitt](https://twitter.com/richlitt)`, since it is a repetitive format and it looks nicer when rendered?

File: README.md (hunk 1 above)
Comment (amor520) [reply README.md]: update 
how does it look now :-D
image
image

**Comment by amor520** [comment]
@RichardLitt  update, thanks :heart:

**Comment by maddhruv** [comment]
Appreciable work done 👏

**Comment by FranzDeCopenhague** [comment]
:( I dropped off the table

**Comment by amor520** [comment]
@FranzDeCopenhague #39

**Comment by maddhruv** [comment]
If someone can volunteer adding all the Github username

**Comment by amor520** [comment]
@maddhruv update
image

**Comment by maddhruv** [comment]
LGTM,
Can be merged

**Comment by RichardLitt** [comment]
Merged. Thank you!

**Comment by obensource** [comment]
WOAH RAD! 👌 

Thanks y'all! 👏👏👏

**Pull request merged**